components/camel-websocket-jsr356/src/main/java/org/apache/camel/jsr356/JSR356Endpoint.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.jsr356;
+
+import org.apache.camel.Consumer;
+import org.apache.camel.Processor;
+import org.apache.camel.Producer;
+import org.apache.camel.impl.DefaultEndpoint;
+import org.apache.camel.spi.UriEndpoint;
+import org.apache.camel.spi.UriParam;
+import org.apache.camel.spi.UriPath;
+
+@UriEndpoint(
+        firstVersion = "2.23.0", scheme = "jsr356", title = "Javax Websocket",
+        syntax = "jsr356:/resourceUri", consumerClass = JSR356Consumer.class, label = "jsr356")
```

Comment:
jsr356 does not looks very nice as component scheme - IMHO

Reply:
Agree but websocket is taken, websocketx? Any good idea?

Reply:
or `websocket-java`?

Reply:
websocket-javax would be probably better than java which does not reference a particular impl but `websocket-jsr356` is at least explicit, let's try to make it merged/mergable with it - to avoid to keep changing the scheme - and once we all agree on the code we can revise it probably. Does it work for everybody this way?

Reply:
absolutely. There is still some work to do on adding the component to kit, but we may work on that after when it will be merged on master after the 2.23.0 release